test/testleakautovar.cpp

```diff
@@ -3045,6 +3046,15 @@ class TestLeakAutoVarStrcpy : public TestFixture {
               "}\n");
         ASSERT_EQUALS("[test.cpp:4]: (error) Memory leak: buf\n", errout.str());
     }
+
+    void doubleFree() {
```

Comment:
this test looks unrelated

Reply:
It certainly is, actually a followup to #5751 / #5823